styles.css

```diff
@@ -12,7 +12,7 @@ h1 {
 .heading-item {
   list-style-type: none;
   font-weight: bold;
-  margin: .5em auto;
+  margin: 0.5em auto;
```

Comment:
In the class they used .5em instead of 0.5em, but for me both looks equally nice=).

Reply:
I actually had there margin -1 at the beggining, but not sure where that got lost

Reply:
lol, my formatter in VS code writes the extra 0.